ras_backstage/controllers/sample_controller.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+from ras_backstage.exception.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def post_sample_file_for_collection_exercise(collection_exercise_id, filename, sample_file_data, survey_type='bres'):
```

Comment:
What's the reason for having `bres` as a default?

Reply:
I think I mistakenly thought that was what the API meant by "survey type". Is it actually "B" / "BI"?

Reply:
I'm not sure :/
Just had a quick look in the survey service and I think this is where we'd specify it was a business survey but I can't find anything that makes it that clear

Reply:
Had another look at the sample service. Looks like every sample thats in it currently has type B.
Also the endpoint is at `samples/{survey_type}/fileupload` the Api.md for sample service doesn't make that very clear